web/src/components/hooks/useIsHelmManaged.js

```diff
@@ -1,39 +0,0 @@
-import { useQuery } from "react-query";
-import { Utilities } from "../../utilities/utilities";
-
-async function fetchIsHelmManaged({
-  accessToken = Utilities.getToken(),
-  apiEndpoint = process.env.API_ENDPOINT,
-  _fetch = fetch,
```

Comment:
not doing this anymore. originally i thought i would unit test this stuff but I think just leaving out any logic is the better path.